osu.Game/Screens/Play/KeyCounterDisplay.cs

```diff
@@ -42,15 +43,15 @@ public override void Add(KeyCounter key)
         [BackgroundDependencyLoader]
         private void load(OsuConfigManager config)
         {
-            config.BindWith(OsuSetting.KeyOverlay, configVisibility);
+            config.BindWith(OsuSetting.KeyOverlay, ConfigVisibility);
         }
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
 
             Visible.BindValueChanged(_ => updateVisibility());
```

Comment:
i'd rename this to `AlwaysVisible` and add xmldoc explaining what it does.

Reply:
You've renamed the other one that's bound to the setting, but I'd also rename `Visible` to `HasReplayLoaded` because that's the only actual case when this is bound to:

https://github.com/ppy/osu/blob/aa1daa0ad50205ea96da37b358e71820c04e41da/osu.Game/Screens/Play/Player.cs#L222

Removes all doubts about what is happening here.

Reply:
i was actually asking for `Visible` to be renamed to `AlwaysVisible` but hey.

Reply:
I think @peppy's way is better here since the key counter needs to be visible when constructed, which will look weird with current names.
 - `alwaysShow = false (default by config)`. 
 - `HasReplayLoaded = false (default by DrawableRuleset)`.

@peppy's way:
 - `configVisibility - false (default by config)`.
 - `AlwaysVisible - true (should commonly be true)`.

Reply:
I don't personally care that much, either's fine at this point compared to the previous version.